modules/runtime/src/home/config/com.enonic.xp.scheduler.cfg

```diff
@@ -0,0 +1,9 @@
+#job.myjob.enabled=true
```

Comment:
it probably should be clean file (for now)
we don't give examples in files, but show default valuse